tests/BlacklistTest.php

```diff
@@ -45,7 +43,7 @@ public function setUp(): void
     }
 
     /** @test */
-    public function it_should_add_a_valid_token_to_the_blacklist()
+    public function itShouldAddAValidTokenToTheBlacklist()
```

Comment:
Aha, that might not be desired

TBH I rarely see PHP projects using the `it_` convention (also quite annoying because it requires the extra `@test` phpdoc; at which point just writing `function test…` feels less overhead anyway)

Reply:
This formatting made for `testdox` format. But since, it handles it alright (I've checked), I think it's OK!